internal/store/store.go

```diff
@@ -114,28 +114,11 @@ func New(stateDir *string) *Store {
 			log.Fatalf("cannot create output directory")
 		}
 
-		stateFile := *stateDir + "/state.json"
-		state, err := ioutil.ReadFile(stateFile)
-
-		if err == nil {
-			err := json.Unmarshal(state, &s)
-			if err != nil {
-				log.Fatalf("invalid initial state: %v", err)
-			}
-		} else if !os.IsNotExist(err) {
+		s.db = jsondb.New(*stateDir, 0600)
+		_, err = s.db.Read("state", &s)
```

Comment:
Should the `"state"` string be extracted to a constant?

Reply:
It wasn't before, but I agree that that would be better. I'll fix it with the next force-push. Thanks!

Reply:
Ha! I just realized I've used `"state"` once and `"store"` the other time. Case in point!